dora/src/semck.rs

```diff
@@ -155,7 +156,7 @@ fn internalck<'ast>(vm: &VM<'ast>) {
             let method = vm.fcts.idx(*method);
             let method = method.read();
 
-            if method.internal && !method.internal_resolved {
+            if method.internal && !method.internal_resolved && method.kind.is_definition() {
```

Comment:
This change means that having `@internal`, but not having decided to use an intrinsic is only considered to be a problem if we don't have an implementation in the source code.